jekyll/_sass/_global-nav.scss

```diff
@@ -0,0 +1,409 @@
+// Values from outer project variables
+///////////////////////////////////////////////////////////////////////////////
+
+$zindex-navbar-fixed:      1030;
+$green: #04AA51;
+$blue-darker: #19639A;
+$white: #FFFFFF;
+$gray: #D8D8D8;
+$black: #161616;
+$transition-duration-medium: 400ms;
+$nav-height: 68px;
+$padding-large-vertical: 11px;
+
+
+// Copy of outer blog-nav, renmaed to global-nav
+///////////////////////////////////////////////////////////////////////////////
+
+.global-nav--component {
+  position: fixed;
+  right: 0;
+  left: 0;
+  z-index: $zindex-navbar-fixed;
+
+  top: 0;
+
+  background-color: $white;
+  box-shadow: 0px 5px 15px 0px rgba(0,0,0,0.15);
+  $shadow-space: 20px;
+
+  @media (max-width: $screen-xs-max) {
+    position: sticky;
+  }
+
+  @media (min-width: $screen-sm) {
+    transition: top $transition-duration-medium ease;
+
+    .nav-hide & {
+      top: calc(-#{$nav-height} - #{$shadow-space});
+    }
+  }
+
+  @media (min-width: $screen-sm) and (max-width: $screen-sm-max) {
+    .mobile-menu-button {
+      display: block;
+    }
+  }
+
+  .icon-bar {
+    background-color: $black;
+  }
+}
+
+.global-nav--lockup {
+  &-link {
+    // not used, yet
+    --link-color: #{$black};
+    --link-color-hover: #{$black};
+
+    &:hover {
+      color: $black;
+    }
+  }
+
+  &-title {
+    font-size: 25.2px;
+    font-weight: 300;
+    letter-spacing: 0.34px;
+    line-height: 24px;
+    margin-left: 9px;
+  }
+}
+
+.global-nav--row {
+  &-mobile {
+    @media (max-width: $screen-sm-max) {
```

Comment:
`$screen-sm-max` = ` 991px`, so this covers tablet, I'm not sure it makes sense for the "mobile" nav to be used for tablet on Docs, especially since the side-nav is still visible on tablet, and click the the menu button only gets you the "start building" button and the link to circleci.com... and actually covers up the side nav: 

![Screen Shot 2021-04-26 at 11 33 06 AM](https://user-images.githubusercontent.com/5113432/116110600-db978900-a683-11eb-80ac-1754d1238f1e.png)

At 991 pixels at the smallest, I think the button and the link can still fit on screen in the nav itself, with no need for menu icon at this size. The button may need to be made narrower.

Reply:
I think I ended up with this way because it's how the header behaves on the blog, and with the search bar the header elements were just too wide in total to fit at the tablet breakpoint.